localization/xliff/vscode-mssql.xlf

```diff
@@ -1587,6 +1587,9 @@
     <trans-unit id="mssql.changeDatabase">
       <source xml:lang="en">Change Database</source>
     </trans-unit>
+    <trans-unit id="mssql.objectExplorerChatWithDatabase">
+      <source xml:lang="en">Chat with this database</source>

```

Comment:
Should I avoid localization changes until we merge to `main`?

Reply:
No reason not to do them now if you remember, but no need to prioritize, IMO.